### ISSUE TITLE
Step 1 Needs Updating for SP Online

### ISSUE THREAD

**Issue by thaeddavid** [MicrosoftDocs/OfficeDocs-SharePoint#1285]
The first step links to a page that doesn't contain steps for SharePoint Online. Would it be possible to have an updated guide to set up BCS for SharePoint Online?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6be8e543-1097-ec1b-204f-37f65fa8b859
* Version Independent ID: 4081bd92-df07-271f-98ad-8e242f24f95f
* Content: [Make an External List from a SQL Azure table with Business Connectivity Services and Secure Store](https://docs.microsoft.com/en-us/sharepoint/make-external-list)
* Content Source: [SharePoint/SharePointOnline/make-external-list.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/live/SharePoint/SharePointOnline/make-external-list.md)
* Service: **sharepoint-online**
* GitHub Login: @kaarins
* Microsoft Alias: **kaarins**

**Comment by scanum** [comment]
@thaeddavid Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by j0rt3g4** [comment]
Hello @thaeddavid , on the Link:
https://support.office.com/article/introduction-to-external-data-676e60e7-d99f-463f-a173-65e9d63538c0

Textually says, on **Know your SharePoint environment** section:

_**Cloud    If you are using SharePoint Online (Plan 2, E3 and E4) in Office 365, then a subset of features are available. For example, external lists, Business Data Web Parts, and the Event framework are supported, but profile pages, actions, and the Connector framework are not. Furthermore, you can only connect to these data sources: WCF Web Services, OData, and SQL Server Azure.**_

Basically this is all the information that is on documentation right now. A way to make it work is thru
"Business Data Web Parts".

With some more development there's a way to use Microsoft Graph:
https://docs.microsoft.com/es-es/sharepoint/dev/spfx/use-aad-tutorial

**Comment by scanum** [comment]
@thaeddavid Thank you for submitting feedback. We understand that this issue has been resolved.
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Comment by JLRishe** [comment]
@scanum This issue hasn't been resolved at all. Step 1 still links to a documentation page that is specifically for SharePoint On-Premises. It even says so at the top:
![image](https://user-images.githubusercontent.com/6964029/70964639-ed504000-20cf-11ea-81ab-deb711731e86.png)
